ddp-workspace/projects/ddp-dsm-ui/src/app/participant-list/participant-list.component.html

```diff
@@ -750,17 +750,18 @@ <h3 class="Color--primary Line--Break">{{additionalMessage}}</h3>
                                     {{tissue[col.participantColumn.name] | noData}}
                                   </ng-container>
                                 </ng-container>
-                                <ng-container *ngIf="col.type === 'ADDITIONALVALUE' && tissue['additionalValuesJson'] != null
-                                && tissue['additionalValuesJson'][col.participantColumn.name] != null; else noTDateTemplate">
+                                <ng-container *ngIf="col.type === 'ADDITIONALVALUE' &&
```

Comment:
Can this code be moved into ts source?  Having logic beyond a single method call inside of the html makes it harder to read, document, and test.

Reply:
Of course!